<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>307</width>
      <height>208</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_serial_port">
         <property name="text">
          <string>Porta:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="box_serial"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>181</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="conectar">
         <property name="text">
          <string>Abrir</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="desconectar">
         <property name="text">
          <string>Fechar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>231</width>
        <height>19</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="connection_status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_velocidade">
      <property name="geometry">
       <rect>
        <x>61</x>
        <y>61</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocidade:  115200 Bauds</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_nome">
              <property name="text">
               <string>Nome:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_preco_bruto">
              <property name="text">
               <string>Pre√ßo Bruto:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_desconto">
              <property name="text">
               <string>Desconto geral:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_desconto_atacado">
              <property name="text">
               <string>Desconto Atacado:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLineEdit" name="nome"/>
            </item>
            <item>
             <widget class="QLineEdit" name="preco_bruto"/>
            </item>
            <item>
             <widget class="QLineEdit" name="desconto"/>
            </item>
            <item>
             <widget class="QLineEdit" name="desconto_atacado"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="atualizar">
            <property name="text">
             <string>Atualizar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="reposicao">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>344</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
